backend/climateconnect_api/serializers/user.py

```diff
@@ -60,19 +67,25 @@ def get_first_name(self, obj):
 
     def get_last_name(self, obj):
         return obj.user.last_name
+    
+    def get_location(self, obj):
+        if obj.location == None:
+            return None
+        return obj.location.name
```

Comment:
Nit: You can simply this to `return obj.location.name if obj.location else None`